client/dashboard/app/queries/domains.ts

```diff
@@ -1,6 +1,15 @@
-import { fetchDomains } from '../../data/domains';
+import { fetchDomains, fetchDomainSuggestions } from '../../data/domains';
+import type { DomainSuggestionQuery } from '../../data/types';
```

Comment:
Super nit: For consistency with other code, I think for queries, we can import from the data layer because it already knows the data layer


```suggestion
import type { DomainSuggestionQuery } from '../../data/domains';
```